queue.tf

```diff
@@ -3,6 +3,7 @@ module "queue-namespace" {
   name                = "${var.product}-servicebus-${var.env}"
   location            = "${var.location}"
   resource_group_name = "${azurerm_resource_group.rg.name}"
+  common_tags         = "${var.common_tags}"
```

Comment:
does this module support it?

Reply:
No, but if we used the azurerm_servicebus_namespace terraform module then tags would be supported.

Reply:
I've updated the terraform services namespace to support this